### ISSUE TITLE
Unable to replace with a dash involved

### ISSUE THREAD

**Issue by tandrewnichols** [75lb/renamer#18]
This command fails:

``` bash
renamer --find '_Small' --replace '-sm'
```

because it thinks `-sm` is another command line flag that it doesn't recognize. I assume this is actually an issue with what library you're using for argument parsing, but I thought you would want to know, as there are plenty of other argument parsers to choose from. Here's the output:

<img width="1374" alt="screen shot 2016-02-22 at 12 16 42 pm" src="https://cloud.githubusercontent.com/assets/2755760/13226181/2b3d07a8-d95e-11e5-9794-66a29848a975.png">

As you can see, it confuses the program a bit.

**Comment by 75lb** [comment]
wtf... thanks, looking into it..

**Comment by 75lb** [comment]
do you have the latest version installed (v0.5.6)?

**Comment by tandrewnichols** [comment]
Yeah, literally just installed like an hour ago.

Unrelated: this is a neat tool. Thanks for building it!

**Comment by 75lb** [comment]
thanks, glad you like it.. i have reproduced the `unknown option` issue - i will address that but your usage guide is fucked up! can't reproduce that on my 10.11.3 Mac..  which shell are you using? Bash?

**Comment by 75lb** [comment]
and which node version are you on?

**Comment by tandrewnichols** [comment]
Bash v4.3.42 and Node v4.2.6. Incidentally, the usage guide actually does _always_ look like that, not just when something blows up.

<img width="1372" alt="screen shot 2016-02-22 at 12 33 53 pm" src="https://cloud.githubusercontent.com/assets/2755760/13226654/90b2ff28-d960-11e5-9b8d-acad17e1f533.png">

**Comment by 75lb** [comment]
hmm, on an almost identical environment i can't reproduce it.. i even fucked around with different fonts and iterm settings.. 

![screen shot 2016-02-22 at 17 40 25](https://cloud.githubusercontent.com/assets/1247980/13226838/6ea4a7f8-d98b-11e5-8b74-75d51e220c56.png)

**Comment by tandrewnichols** [comment]
I'm using the font "Meslo LG S Regular for Powerline" (which is a powerline font, obviously), so . . . it could be me.

**Comment by ismyrnow** [comment]
I get similar output using the standard command prompt on Windows 7, and node 0.10 (not my typical environment, but possibly worth mentioning). The font there is "Terminal".

**Comment by 75lb** [comment]
Fixed and released in renamer v0.6.1.

The "Unable to replace with a dash involved" issue is resolved within the command-line-args module, which renamer uses.

See [here](https://github.com/75lb/command-line-args#ambiguous-values) for a description how to avoid ambiguous values. In summary, you use `--option=value` notation, for example:

```
renamer --find '_Small' --replace='-sm'
```
